libraries/entities-renderer/src/RenderableTextEntityItem.cpp

```diff
@@ -324,7 +325,7 @@ void entities::TextPayload::render(RenderArgs* args) {
     glm::vec2 bounds = glm::vec2(dimensions.x - (textRenderable->_leftMargin + textRenderable->_rightMargin), dimensions.y - (textRenderable->_topMargin + textRenderable->_bottomMargin));
     textRenderer->draw(batch, textRenderable->_leftMargin / scale, -textRenderable->_topMargin / scale, bounds / scale, scale,
                        textRenderable->_text, textRenderable->_font, textColor, effectColor, textRenderable->_effectThickness, textRenderable->_effect,
-                       textRenderable->_unlit, forward);
+                       textRenderable->_alignment, textRenderable->_unlit, forward);
```

Comment:
the number of things has definitely grown since we started adding more effects to text, and it all does need to get passed down through several classes/function calls, but I don't think we need to do this right now